docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/performance-navigation-timing/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Performance Navigation Timing"
+date: "2023-05-10"
+sidebar_position: 14500
+---
+
+```mdx-code-block
+import Block5966 from "@site/docs/reusable/javascript-tracker-release-badge-v3/_index.md"
+
+<Block5966/>
+```
+
+:::note
+The plugin is available since version 3.10 of the tracker.
+:::
```

Comment:
Could we add the first few paragraphs and the diagram from [the README of the plugin](https://github.com/snowplow/snowplow-javascript-tracker/blob/master/plugins/browser-plugin-performance-navigation-timing/README.md)? I think it would be useful to clarify what the plugin does.